concurrent/src/main/scala/tofu/concurrent/ContextT.scala

```diff
@@ -71,7 +71,7 @@ object ContextT extends ContextTInstances {
 
 }
 
-trait ContextTInstances extends ContextTInstancesQ
+trait ContextTInstances extends ContextTInstancesP
```

Comment:
I think it's better to use meanigful name, like ContextTInstancesPrimary or something else.
Source code readability matters here, reader should be able to understand why some instances are in another trait.

Scaladoc is another option

Reply:
It's just Z, Y, X, .... here, just like 1, 2, 3... in other instance ladders

Reply:
We can change this after kernelization